README-JP.md

```diff
@@ -62,6 +62,17 @@ cd _rel/emqx && ./bin/emqx console
 
 *EMQ X* ブローカーを起動したら、ブラウザで http://localhost:18083 にアクセスしてダッシュボードを表示できます。
 
+### ダイアライザー
+##### すべてのアプリを分析するには
```

Comment:
This is confusing because you do not mention what will be analyzed. Also, `アプリ` should not be used in a formal context. I would say `アプリケーションの型情報を解析する`

Reply:
This is confusing because you do not mention what will be analyzed. Also, アプリ should not be used in a formal context. I would say アプリケーションの型情報を解析する